utils/socket.c

```diff
@@ -52,13 +52,20 @@ int socket_listen(int fd, struct sockaddr_un *addr)
 	return 0;
 }
 
-/* Send a single option to the agent through its socket */
-int socket_send_option(int fd, enum uftrace_dopt opt, void *value, size_t size)
+/**
+ * socket_send_option - send a single option to the agent through its socket
+ * @fd - agent socket file descriptor
+ * @opt - option to send
+ * @data - data load for the option
+ * @size - data size
+ * @return - status code, negative for error
+ */
+int socket_send_option(int fd, enum uftrace_agent_msg opt, void *data, size_t size)
```

Comment:
Maybe we can call it `socket_send_message()` or `uftrace_socket_send_message()`.

Reply:
OK, same with all other socket utils that are used in libmcount/live?